set-definitions/src/main/java/eu/europeana/set/definitions/model/impl/BaseUserSet.java

```diff
@@ -371,4 +215,34 @@ public boolean isPublic() {
     public boolean isPublished() {
 	return VisibilityTypes.PUBLISHED.getJsonValue().equals(getVisibility());
     }
+
+    /**
+     * if List<String>subject contains an entity reference
+     * referring to the set then it is a EntityBestItemSet
+     *
+     * @return true if it is an EntityBestItemSet
+     */
+    public boolean isEntityBestItemSet() {
```

Comment:
there is a method isEntityBestItemSet in the service with the correct implementation. I think we can remove this one.

Reply:
done